diff --git a/src/base/ftver.rc b/src/base/ftver.rc
index eb6090ecc..289781599 100644
--- a/src/base/ftver.rc
+++ b/src/base/ftver.rc
@@ -45,7 +45,7 @@ BEGIN
       VALUE "FileVersion",      FT_VERSION_STR
       VALUE "ProductName",      "FreeType"
       VALUE "ProductVersion",   FT_VERSION_STR
-      VALUE "LegalCopyright",   "\251 2000-2022 The FreeType Project www.freetype.org. All rights reserved."
+      VALUE "LegalCopyright",   L"\251 2000-2022 The FreeType Project www.freetype.org. All rights reserved."
       VALUE "InternalName",     "freetype"
       VALUE "OriginalFilename", FT_FILENAME
     END
